Movement.cpp

```diff
@@ -8,145 +8,87 @@
 
 Movement::Movement() {}
 
-void Movement::advance(const double desire, const double desireUltrasonic) {
-  eCount1 = 0;
-  eCount2 = 0;
+void Movement::advanceP(const double desire) {
```

Comment:
advanceP? What does P stand for.

Reply:
Also, I'll check the logic of this function once you attend the comments on the BNO file